admin/data-collection.rst

```diff
@@ -15,27 +15,22 @@ Every node can have many data collection items configured (see
 set of threads dedicated to data collection, called `Data Collectors`, used to
 gather information from the nodes according to :term:`DCI` configuration. You
 can control how many data collectors will run simultaneously, by changing server
-configuration parameter ``NumberOfDataCollectors``.
-
-All configured DCIs are checked for polling requirement every second
-Main information about node(:guilabel:`Object Details`) can be supplemented with
-DCI information displayed as text(last value) on :guilabel:`Object Details`->
-:guilabel:`Overview` page or in graph way on :guilabel:`Object
-Details`->:guilabel:`Performance` tab.
-
-DCI representation in text way can be configured on
-:ref:`dci-othe-options-label`. Next will be described only graph DCI
-representation configuration on :guilabel:`Performance` tab of :guilabel:`Object
-Details`.
-
-Multiple DCIs can be grouped in one graph. To group them use the same group name
-in "Group" field. and
-if DCI needs to be polled, appropriate polling request is placed into internal
+configuration parameter ``ThreadPool.DataCollector.MaxSize``.
+
+All configured DCIs are checked for polling requirement every second.
+Main information about node ( see :guilabel:`Overview` tab ) can be granulated with
+node DCI capability information displayed in :guilabel:`Overview`-> :guilabel:`Capabilities` 
+section as well as last values in :guilabel:`Data Collection` tab view.
+
+Supplimentary DCI representation can be configured in
```

Comment:
Chapter separation looks odd. 

I would separate chapter about displaying data from the chapter that describes how data collection is done.

Like:

Rework of text about Capabilities. 

Supplimentary DCI representation can be configured in :ref:`dci-othe-options-label` or :ref:`dci-performance-tab`. 

All configured DCIs are checked for polling requirement every second. If DCI needs to be polled, appropriate polling request is placed into internal ...